### PR TITLE
Init Maps Dashboard Plugin Repository

### LINKED ISSUE #2: Initialize new plugin and update README
- Build new plugin
- Update README
#1

### PR DESCRIPTION
**Pull request by DDG18**
### Description
- Auto generate the skeleton code for the Maps Dashboard plugin
- Add Readme
 
### Issues Resolved
 #2 

### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by VijayanB** [comment]
Can you add snapshot/image from your local host on how does this look ?

**Comment by VijayanB** [comment]
Also, create new issue [here](https://github.com/VijayanB/maps-dashboards-plugin/issues) and link it in PR description

**Comment by DDG18** [comment]
Screen Shot 2022-05-16 at 6 21 14 PM
Screen Shot 2022-05-16 at 6 22 02 PM


> Can you add snapshot/image from your local host on how does this look ?

File: opensearch_dashboards.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "mapsDashboardsPlugin",
```
Comment (VijayanB) [inline opensearch_dashboards.json:2]: mapsDashboards

Comment (VijayanB) [inline opensearch_dashboards.json:2]:
```suggestion
  "id": "mapsDashboards",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "mapsDashboardsPlugin",
```
Comment (VijayanB) [inline package.json:2]:
```suggestion
  "name": "maps-dashboards",
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  "version": "0.0.0",
```
Comment (VijayanB) [inline package.json:3]:
```suggestion
  "version": "1.0.0",
```

**Comment by DDG18** [comment]
> Can you recreate with plugin id as "mapsExplorerDashboards" ? You can refer this plugin as an example: https://github.com/opensearch-project/observability/tree/main/dashboards-observability We also want on left side as "Maps Explorer".

Hi Vijay, thanks for your comments. I just pushed the commit with changes you are asking to the PR.

**Comment by DDG18** [comment]
Attached some screenshots.

Screen Shot 2022-05-17 at 9 35 19 AM
Screen Shot 2022-05-17 at 9 36 01 AM

**Pull request merged**